roles/docker/tasks/main.yml

```diff
@@ -28,7 +28,20 @@
     state: started
 
 # Install docker-compose which also installs the docker python library required by some ansible docker modules
-- name: install docker-compose
+- name: install docker
   pip:
-    name: "docker-compose"
+    name: "docker"
     state: present
+
+- name: check ssl_match_hostname
+  stat:
+    path: /usr/lib/python2.7/dist-packages/backports/ssl_match_hostname
```

Comment:
We shouldn't use Python 2 anymore since it's deprecated. Any way to use Python 3?

Reply:
Ansible docker_container and docker_image is using python 2. I tried figuring out how to change that but was unsuccessful.

Reply:
I think it's due to the environment you were testing is using Python 2. Base on Ansible doc, both Python 2 and 3 should work: https://docs.ansible.com/ansible/latest/scenario_guides/guide_docker.html#requirements